src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict

```diff
@@ -55,13 +55,13 @@
 			<string>u</string>
 			<key>zero</key>
 			<!-- XFLD: Plural form for Zero -->
-			<string>Last encounter: today</string>
+			<string>%u days since last exposure</string>
 			<key>one</key>
 			<!-- XFLD: Plural form for One -->
-			<string>Last encounter: yesterday</string>
+			<string>%u day since last exposure</string>
```

Comment:
I guess this should also have non-breakable spaces

Reply:
Same goes for the German equivalent.

Reply:
done